src/AWS.Deploy.CLI/Utilities/CommandLineWrapper.cs

```diff
@@ -73,6 +73,7 @@ public class CommandLineWrapper : ICommandLineWrapper
                 processStartInfo.UseShellExecute = _useSeparateWindow;
             }
 
+            UpdateEnvironmentVariables(processStartInfo, environmentVariables);
```

Comment:
I wonder how can we make unbreakable as a thought exercise.

Reply:
I dont know about unbreakable, but we could abstract the Process class and add the telemetry stuff in there. Anything that spins up a new process would have the telemetry stuff in there.